src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/PublicationSummary.test.tsx

```diff
@@ -0,0 +1,332 @@
+      title: 'Release 2',
+      publicationId: testPublication.id,
+      permissions: {
+        canUpdateRelease: false,
+      },
+      approvalStatus: 'Approved',
+    } as MyRelease,
+  ];
+
+  const testMethodologies: MyPublicationMethodology[] = [
+    {
+      owner: true,
+      methodology: {
+        amendment: false,
+        id: '1234',
+        latestInternalReleaseNote: 'this is the release note',
+        previousVersionId: 'lfkjdlfj',
+        published: '2021-06-08T09:04:17.9805585',
+        slug: 'meth-1',
+        status: 'Approved',
+        title: 'I am a methodology',
+        owningPublication: {
+          id: 'p1',
+          title: 'Publication title',
+        },
+        permissions: {
+          canApproveMethodology: false,
+          canUpdateMethodology: false,
+          canDeleteMethodology: false,
+          canMakeAmendmentOfMethodology: false,
+          canMarkMethodologyAsDraft: false,
+        },
+      },
+      permissions: {
+        canDropMethodology: false,
+      },
+    },
+  ];
+
+  const testExternalMethodology: MyPublication['externalMethodology'] = {
+    title: 'External methodology',
+    url: 'https://example.com',
+  };
+
+  test('renders correctly with a publication with no releases, methodologies, contact details or permissions', async () => {
+    render(
+      <MemoryRouter>
+        <PublicationSummary
+          publication={testPublication}
+          topicId={testTopicId}
+          onChangePublication={noop}
+        />
+      </MemoryRouter>,
+    );
+
+    expect(screen.getByText('No team name')).toBeInTheDocument();
+    expect(screen.getByText('No contact name')).toBeInTheDocument();
+
+    expect(screen.getByText('No releases created')).toBeInTheDocument();
+
+    expect(screen.getByText('Methodologies')).toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('button', {
+        name: 'Create methodology',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(
+      screen.queryByRole('button', {
+        name: 'Link to an externally hosted methodology',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(
+      screen.queryByText('Manage this publication', {
+        selector: 'a',
+      }),
+    ).not.toBeInTheDocument();
+
+    expect(
+      screen.queryByText('Create new release', {
+        selector: 'a',
+      }),
+    ).not.toBeInTheDocument();
+  });
+
+  test('renders correctly with team and contact details', async () => {
+    render(
+      <MemoryRouter>
+        <PublicationSummary
+          publication={{
+            ...testPublication,
+            contact: testContact,
+          }}
+          topicId={testTopicId}
+          onChangePublication={noop}
+        />
+      </MemoryRouter>,
+    );
+
+    expect(screen.queryByText('No team name')).not.toBeInTheDocument();
+    expect(screen.queryByText('No contact name')).not.toBeInTheDocument();
+
+    expect(
+      screen.getByText(testContact.teamName as string),
```

Comment:
Could specify the exact specific contact details to search for instead of using `as string` - would do this for the other assertions too if so

Reply:
Done